cmd/nerdctl/namespace.go

```diff
@@ -114,7 +125,130 @@ func namespaceLsAction(cmd *cobra.Command, args []string) error {
 			numVolumes = len(volEnts)
 		}
 
-		fmt.Fprintf(w, "%s\t%d\t%d\t%d\n", ns, numContainers, numImages, numVolumes)
+		labels, err := client.NamespaceService().Labels(ctx, ns)
+		if err != nil {
+			return err
+		}
+		for k, v := range labels {
+			labelStrings = append(labelStrings, strings.Join([]string{k, v}, "="))
+		}
+		sort.Strings(labelStrings)
+		fmt.Fprintf(w, "%s\t%d\t%d\t%d\t%v\t\n", ns, numContainers, numImages, numVolumes, strings.Join(labelStrings, ","))
 	}
 	return w.Flush()
 }
+
+func newNamespaceRmCommand() *cobra.Command {
+	namespaceRmCommand := &cobra.Command{
+		Use:           "remove [OPTIONS] NAMESPACE [NAMESPACE...]",
+		Aliases:       []string{"rm"},
+		Short:         "Remove one or more namespaces",
+		RunE:          namespaceRmAction,
+		SilenceUsage:  true,
+		SilenceErrors: true,
+	}
+	namespaceRmCommand.Flags().BoolP("cgroup", "c", false, "delete the namespace's cgroup")
+	return namespaceRmCommand
+}
+
+func namespaceRmAction(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		return errors.New("please specify a namespace")
+	}
+	var exitErr error
+	client, ctx, cancel, err := newClient(cmd)
+	if err != nil {
+		return err
+	}
+	defer cancel()
+	opts, err := deleteOpts(cmd)
+	if err != nil {
+		return err
+	}
+	namespaces := client.NamespaceService()
+
+	for _, target := range args {
+		if err := namespaces.Delete(ctx, target, opts...); err != nil {
```

Comment:
We have to delete containers, images, and volumes before deletion of the namespace

Reply:
It should be also documented that we may have dangling objects (e.g., containers, images, snapshots) in the daemon's database when an object was created during we remove the namespace (IIRC)

Reply:
i try to delte a not null namespace,it will show error
```
ATA[0000] unable to delete default: namespace "default" must be empty, but it still has images, blobs, containers, snapshot-overlayfs: failed precondition
```

Reply:
I need  time to find out it

Reply:
`--force` can be added in a separate PR.

Reply:
ok